packages/gatsby/cache-dir/load-directly-or-404.js

```diff
@@ -36,7 +36,10 @@ export function getRedirectUrl(path) {
 export default function(resources, path, replaceOnSuccess = false) {
   return new Promise((resolve, reject) => {
     const url = getRedirectUrl(path)
-    if (!url) return reject(url)
+    if (!url) {
+      reject(url)
+      return
```

Comment:
This may as well just return the rejection, right?

Reply:
having them separate fixes inconsistent return  eslint rule

Reply:
Thought it might be because of the inline return 🤷‍♂️ Yeah, this is fine either way!